src/DssSpell.t.base.sol

```diff
@@ -472,15 +558,17 @@ contract DssSpellTestBase is Config, DSTest, DSMath {
             }
 
             {
-            (,uint256 mat) = spotter.ilks(ilk);
+            (address pip, uint256 mat) = spotter.ilks(ilk);
             // Convert BP to system expected value
             uint256 normalizedTestMat = (values.collaterals[ilk].mat * 10**23);
-            if (values.collaterals[ilk].lerp) {
-                assertTrue(mat <= normalizedTestMat, concat("TestError/vat-lerping-mat-", ilk));
-                assertTrue(mat >= RAY && mat <= 300 * RAY, concat("TestError/vat-mat-range-lerp-", ilk));
-            } else {
-                assertEq(mat, normalizedTestMat, concat("TestError/vat-mat-", ilk));
-                assertTrue(mat >= RAY && mat < 10 * RAY, concat("TestError/vat-mat-range-", ilk));    // cr eq 100% and lt 1000%
+            if (pip != address(0)) {
```

Comment:
This code differs a bit from the goerli one. If we can standardize both would be better.